output/plugins/additional/xml/additional.pythoncode

```diff
@@ -199,7 +199,8 @@ Python code generation written by
     	#ifnotnull $visited_color
     	@{ #nl self.$name.SetVisitedColour( $visited_color ) @}
 	</template>
-	<template name="evt_connect_OnHyperlink">self.$name.Bind( wx.EVT_HYPERLINK, #handler )</template>
+    <template name="include">import wx.adv</template>
```

Comment:
Can you use Tab instead of four spaces here?

Reply:
Yep, no worries..